src/concrete/ml/onnx/convert.py

```diff
@@ -191,12 +190,11 @@ def get_equivalent_numpy_forward_from_onnx(
         "eliminate_unused_initializer",
     ]
     equivalent_onnx_model = onnxoptimizer.optimize(onnx_model, onnx_passes)
-    checker.check_model(equivalent_onnx_model)
```

Comment:
redundant checks

Reply:
I don't believe this is redundant, we are checking a "new" onnx model here (which has been _optimized_), so I think it's better to check that it is still correct

Reply:
added back